protocols/grpc/agent.proto

```diff
@@ -434,3 +435,20 @@ message StringUser {
 	string gid = 2;
 	repeated string additionalGids = 3;
 }
+
+message CopyFileRequest {
+	// Path is the destination path in the guest.
```

Comment:
Thanks for adding the comment. I think it could be made clearer though that the path must be:

- absolute and canonical.
- below `/run`.